lib/pxconfig.js

```diff
@@ -36,7 +36,7 @@ class PxConfig {
             ['MODULE_MODE', 'moduleMode'], ['FIRST_PARTY_ENABLED', 'firstPartyEnabled'], ['ADDITIONAL_ACTIVITY_HANDLER', 'additionalActivityHandler'], ['ENRICH_CUSTOM_PARAMETERS', 'enrichCustomParameters'],
             ['TESTING_MODE', 'testingMode'], ['WHITELIST_EXT', 'whitelistExt'], ['BYPASS_MONITOR_HEADER', 'bypassMonitorHeader'], ['ADVANCED_BLOCKING_RESPONSE', 'advancedBlockingResponse'],
             ['TELEMETRY_COMMAND_HEADER', 'telemetryCommandHeader'], ['CUSTOM_TEMPLATE_ROOT', 'customTemplateRoot'], ['CUSTOM_TEMPLATE_DATA', 'customTemplateData'], ['FILTER_BY_IP', 'filterByIP'], ['FILTER_BY_USERAGENT', 'filterByUserAgent'], ['FILTER_BY_METHOD', 'filterByMethod'],
-            ['EXTERNAL_ACTIVITIES', 'externalActivities'], ['PXHD_SECURE', 'pxhdSecure']];
+            ['EXTERNAL_ACTIVITIES', 'externalActivities'], ['PXHD_SECURE', 'pxhdSecure'], ['CUSTOM_LOGGER', 'customLogger']];
```

Comment:
I think we can remove this change (and the default value below) now that we are consuming `params` directly?